TileFormats/Instanced3DModel/README.md

```diff
@@ -98,19 +121,51 @@ In either case, `header.gltfByteLength` contains the length of the glTF field in
 
 The `instances` field immediately follows the `glTF` field (which may be omitted when `header.gltfByteLength` is `0`).
 
-The `instances` field contains `header.instancesLength` of tightly packed instances.  Each instance has three fields:
+The `instances` field contains `header.instancesLength` of each instance field which are stored in groups of each field.
+
+Each instance has the following fields:
+
+| Field name | Data type | Description | Required |
+| --- | --- | --- | --- |
+| `position.x` | `uint16` | The x-coordinate in quantized Cartesian coordinates. | `yes` |
+| `position.y` | `uint16` | The y-coordinate in quantized Cartesian coordinates. | `yes` |
+| `position.z` | `uint16` | The z-coordinate in quantized Cartesian coordinates. | `yes` |
+| `encodedAxis.x` | `uint` | The first component of the oct-encoded `axis` vector. | `yes` |
+| `encodedAxis.y` | `uint` | The second component of the oct-encoded `axis` vector. | `yes` |
+| `encodedAngle` | `uint` | The angle to rotate about the axis from `0` to `2π` scaled to fill a single byte. | `yes` |
```

Comment:
We have a precision of ~1.4 degrees for rotation with axis-angle. @bagnell what do you think? If we need more precision, is it better to make the angle a `uint16` and get ~0.005 degrees or to go back to two vectors?